### ISSUE TITLE
[Bug]: Unable to add Qingping Air Monitor Lite CGDN1

### ISSUE THREAD

**Issue by klymenko4** [custom-components/ble_monitor#1147]
### What happened?

Hello, here is some log after I added MAC to ble monitor.
Device is connected to Qingping + and to Xiaomi home also.
Hope this log will be enough. 

### Sensor type

Qingping Air Monitor Lite CGDN1

### Relevant log output

```shell
2023-03-15 21:26:29.311 INFO (MainThread) [custom_components.ble_monitor] Attention! Option report_unknown is enabled for Qingping sensors, be ready for a huge output
2023-03-15 21:26:29.311 INFO (MainThread) [custom_components.ble_monitor] Attention! Option report_unknown is enabled for sensor with id(s): ['7CC294C9EBB8']
2023-03-15 21:26:30.191 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -64, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100ba0112040a000a0013026f02
2023-03-15 21:26:30.192 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xba\x01\x12\x04\n\x00\n\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:30.205 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:30.444 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:30.706 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:31.194 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:31.216 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -64, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100ba0112040a000a0013026f02
2023-03-15 21:26:31.216 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xba\x01\x12\x04\n\x00\n\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:31.536 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -61, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100ba0112040a000a0013026f02
2023-03-15 21:26:31.536 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xba\x01\x12\x04\n\x00\n\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:31.847 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -68, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100ba0112040a000a0013026f02
2023-03-15 21:26:31.847 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xba\x01\x12\x04\n\x00\n\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:31.848 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -61, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100ba0112040a000a0013026f02
2023-03-15 21:26:31.848 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xba\x01\x12\x04\n\x00\n\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:32.235 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:32.489 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:32.727 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:32.975 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:33.211 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -61, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100bb0112040a000a0013026f02
2023-03-15 21:26:33.211 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xbb\x01\x12\x04\n\x00\n\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:33.539 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -62, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100bb0112040a000a0013026f02
2023-03-15 21:26:33.539 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xbb\x01\x12\x04\n\x00\n\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:34.209 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:34.451 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:34.701 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:35.069 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:35.179 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:35.530 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -64, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100bb0112040a000b0013026f02
2023-03-15 21:26:35.530 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xbb\x01\x12\x04\n\x00\x0b\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:35.851 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -64, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100bb0112040a000b0013026f02
2023-03-15 21:26:35.852 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xbb\x01\x12\x04\n\x00\x0b\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:36.194 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -61, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d100bb0112040a000b0013026f02
2023-03-15 21:26:36.194 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd1\x00\xbb\x01\x12\x04\n\x00\x0b\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
2023-03-15 21:26:36.197 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: []manufacturer specific data: [b'\x0f\xff\x8f\x03\n\x10=\x0c\x00\xb6\xeb\xc9\x94\xc2|\x81']local name: UUID16: None,UUID128: None
2023-03-15 21:26:37.216 INFO (Thread-2) [custom_components.ble_monitor.ble_parser.qingping] BLE ADV from UNKNOWN Qingping DEVICE: RSSI: -64, MAC: 7C:C2:94:C9:EB:B8, ADV: 1b16cdfd0824b6ebc994c27c0104d200bb0112040a000b0013026f02
2023-03-15 21:26:37.217 INFO (Thread-2) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID 7cc294c9ebb8: service data: [b'\x1b\x16\xcd\xfd\x08$\xb6\xeb\xc9\x94\xc2|\x01\x04\xd2\x00\xbb\x01\x12\x04\n\x00\x0b\x00\x13\x02o\x02']manufacturer specific data: []local name: UUID16: None,UUID128: None
```

**Comment by Ernst79** [comment]
Your device has a new device id `0x24`, the old device id for the CGDN1 was `0x0E`. I'll add it.

**Comment by Ernst79** [comment]
Fixed in 11.8.1
